src/codegen/protos/nidaqmx.proto

```diff
@@ -9931,38 +9934,6 @@ message TristateOutputTermResponse {
   int32 status = 1;
 }
 
-message UnregisterDoneEventRequest {
-  nidevice_grpc.Session task = 1;
-}
-
-message UnregisterDoneEventResponse {
-  int32 status = 1;
-}
-
-message UnregisterEveryNSamplesEventRequest {
-  nidevice_grpc.Session task = 1;
-  oneof every_n_samples_event_type_enum {
-    EveryNSamplesEventType every_n_samples_event_type = 2;
-    int32 every_n_samples_event_type_raw = 3;
-  }
-}
-
-message UnregisterEveryNSamplesEventResponse {
-  int32 status = 1;
-}
-
-message UnregisterSignalEventRequest {
-  nidevice_grpc.Session task = 1;
-  oneof signal_id_enum {
-    Signal2 signal_id = 2;
-    int32 signal_id_raw = 3;
-  }
-}
-
-message UnregisterSignalEventResponse {
-  int32 status = 1;
-}
```

Comment:
Your nidaqmx.proto is out of date. You need to merge main into your grpc-device branch and rebuild.